modules/heat_conduction/doc/content/source/kernels/ADHeatConduction.md

```diff
@@ -0,0 +1,18 @@
+<!-- MOOSE Documentation Stub: Remove this when content is added. -->
+
+# ADHeatConduction
+
+!alert construction title=Undocumented Class
+The HeatConduction has not been documented. The content contained on this page
+includes the basic documentation associated with a MooseObject; however, what is contained is
+ultimately determined by what is necessary to make the documentation clear for users.
+
+!syntax description /Kernels/ADHeatConduction
```

Comment:
Unfortunately, for now this needs to look like:
```c++
!syntax description /ADKernels/ADHeatConduction<RESIDUAL>
```

Reply:
`/ADKernels/` or `/Kernels/`?

Reply:
The former. It matches the input file syntax, e.g. the `ADHeatConduction` sub-block will be put inside the `ADKernels` block

Reply:
Gotcha